Est/Handler/XmlFile.php

```diff
@@ -46,8 +46,8 @@ protected function _apply() {
 
 
         $dom = new DOMDocument();
-        $dom->preserveWhiteSpace = true;
-        $dom->formatOutput = false;
+        $dom->preserveWhiteSpace = false;
```

Comment:
As I wrote in my pull request comment:

The following lines:

`$dom->preserveWhiteSpace = false;`
`$dom->formatOutput = true;`

were changed to make XML file human-readable.

---------

After adding new nodes they were all in one line without any tabulations and new lines.
What's wrong with such configuration?